jac-uk/admin
# #1841 Select applications for character checks by application status

## What's included?
Adds dropdown selections for `stage` and `status`.
Using `availableStatuses` so should slot in and be improved by [this pr](https://github.com/jac-uk/admin/pull/1931).

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Navigate to an exercise with character checks 
- Check the available stages and status work to filter the table contents
- find any flaws or issues within this

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
![image](https://user-images.githubusercontent.com/44227249/225716294-e1563fd0-d856-4c86-9709-1e35d184a1ea.png)

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
